config/freebsd

```diff
@@ -18,19 +18,26 @@
 #
 # See BlankForm in this directory for details.
 
-# The default compiler is `gcc'
+# The default compiler is 'cc'
 if test "X-" = "X-$CC"; then
-  CC=gcc
-  CC_BASENAME=gcc
+  CC=cc
+  CC_BASENAME=cc
```

Comment:
This just changes the command configure uses to detect the brand of compiler—gcc, clang, Intel—right? Looks fine to me.